modules/broker-amqp/src/main/java/io/ballerina/messaging/broker/amqp/codec/frames/QueueDelete.java

```diff
@@ -101,16 +102,22 @@ public void handle(ChannelHandlerContext ctx, AmqpConnectionHandler connectionHa
             } catch (ValidationException e) {
                 LOGGER.debug("Queue delete validation failure", e);
                 ctx.writeAndFlush(new ChannelClose(getChannel(),
-                                                   ChannelException.PRECONDITION_FAILED,
-                                                   ShortString.parseString(e.getMessage()),
-                                                   CLASS_ID,
-                                                   METHOD_ID));
+                        ChannelException.PRECONDITION_FAILED,
```

Comment:
The previous indentation is correct IMO. Shall we revert back?

Reply:
Fixed in b30ae14